@creditkarma/mimic-cli is missing @creditkarma/mimic-graphql dependency

**Describe the bug**
A clear and concise description of what the bug is.

**To Reproduce**
Steps to reproduce the behavior:
1. `npm  install -g @creditkarma/mimic-cli`
2. `mimic --help`
3. `Error: Cannot find module '@creditkarma/mimic-graphql`

**Expected behavior**
$ mimic --help
  Usage: mimic [options] <file.mimic> [-r|--request] <request.json>
  Options:

    -h, --help     output usage information
    -a, --all      enable all services
    -r, --request  switch to client mode and send request
    -V, --version  output the version number

**Screenshots/Logs**
`mimic --help`
`internal/modules/cjs/loader.js:583
    throw err;
    ^
Error: Cannot find module '@creditkarma/mimic-graphql'
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:581:15)
    at Function.Module._load (internal/modules/cjs/loader.js:507:25)
    at Module.require (internal/modules/cjs/loader.js:637:17)
    at require (internal/modules/cjs/helpers.js:20:18)
    at Object.<anonymous> (/Users/volodymyr.ladnik/.nvm/versions/node/v10.8.0/lib/node_modules/@creditkarma/mimic-cli/dist/index.js:4:25)
    at Module._compile (internal/modules/cjs/loader.js:689:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:700:10)
    at Module.load (internal/modules/cjs/loader.js:599:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:538:12)
    at Function.Module._load (internal/modules/cjs/loader.js:530:3)`

**Desktop/Server (please complete the following information):**
 - OS: OSX
 - Node version: v10.8.0
 - Version 2.0.3

**Additional context**
Can be resolved by installing `@creditkarma/mimic-graphql` package explicitly

--- a/packages/mimic-cli/package.json
+++ b/packages/mimic-cli/package.json
@@ -23,6 +23,7 @@
   },
   "dependencies": {
     "@creditkarma/mimic-core": "^2.0.3",
+    "@creditkarma/mimic-graphql": "^2.0.3",
     "@creditkarma/mimic-rest": "^2.0.3",
     "@creditkarma/mimic-thrift": "^2.0.3"
   },

